python/core/auto_generated/effects/qgsblureffect.sip.in

```diff
@@ -63,16 +63,68 @@ Sets blur level (strength)
 .. seealso:: :py:func:`blurMethod`
 %End
 
-    int blurLevel() const;
+    double blurLevel() const;
 %Docstring
-Returns the blur level (strength)
+Returns the blur level (radius)
```

Comment:
Ideally add seealso links to the blur unit and map scale methods here too -- it's a nice visual queue for api end users that it's possible to set the units and how to do so.